debugging-webpack/README.md

```diff
@@ -0,0 +1,52 @@
+# Debugging webpack application
```

Comment:
Same goes here, as this guide really isn't about debugging webpack applications, but web apps that has been bundled via webpack, and this is a guide on how you configure the generated sourcemaps for webpack to be compatible with VS Code.